src/screens/Shared/EditLocation.js

```diff
@@ -52,6 +52,19 @@ class EditLocation extends Component {
     }
   };
 
+  displayCoordinates = async coordinates => {
```

Comment:
Same as the displayAddress comment-- I'd probably verb this function name too. (setDisplayCoordinates?)